modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2669,6 +2669,89 @@ TEST_P(Test_ONNX_layers, YOLOv7)
     normAssertDetections(refClassIds, refScores, refBoxes, keep_classIds, keep_confidences, keep_boxes);
 }
 
+
+TEST_P(Test_ONNX_layers, YOLOv5m)
+{
+    std::string weightPath = _tf("models/yolov5m.onnx", false);
```

Comment:
You need to create PR with the model link in opencv_extra. PR should be created with the same branch name. Test models are downloaded by this script and listedt inside of it: https://github.com/opencv/opencv_extra/blob/4.x/testdata/dnn/download_models.py

Reply:
And there is already YOLOv5n, no need for 5m version: https://github.com/opencv/opencv/blob/0e151e3c8837441c961d7f4788c733d634cd4964/modules/dnn/perf/perf_net.cpp#L275

Reply:
@are they the same though?

Reply:
I think they differ in number of repeated blocks or something.

Reply:
@dkurt  Ar you suggesting to use weights of yolox5n  for this test or to remove the test entirely?

Reply:
Accuracy test is a good addition. For now there is only performance test for YOLOv5 as I see. So I suggest replace YOLOv5m to YOLOv5n.